From bfaa94316b3f31edffb76e4831520a05b8cb8f6e Mon Sep 17 00:00:00 2001
From: mldn <mldn@163.com>
Date: Thu, 19 Nov 2020 18:23:45 +0800
Subject: [PATCH] Change Admin.java

---
 Admin.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Admin.java b/Admin.java
index 6f7f237..ccf1b84 100644
--- a/Admin.java
+++ b/Admin.java
@@ -1,4 +1,5 @@
 import java.util.List ;
 public class Admin{
 	private List<Emp> emps ;
+	private String aid ;
 }
\ No newline at end of file
-- 
2.29.2.windows.2

